### PR TITLE
Fix mistakes in the Chromium data for DOMMatrix/DOMMatrixReadOnly

### PR DESCRIPTION
**Pull request by foolip**
First, remove the GeometryInterfaces flag, removed over 2 years ago:
https://chromium.googlesource.com/chromium/src/+/3e51c4becac41fe61dbd02a68bd875cc4a340457

Lint errors made it apparent that a lot of the versions were wrong.

The support data here comes mainly from the following:
https://github.com/mdn/browser-compat-data/pull/1617
https://github.com/mdn/browser-compat-data/pull/3644
https://github.com/mdn/browser-compat-data/pull/3743
https://github.com/mdn/browser-compat-data/pull/4902
https://github.com/mdn/browser-compat-data/pull/4936

The root mistake was looking at commits which added/updated these APIs
while it was still behind a flag, and some BCD entries using those
releases without documenting it as behind a flag.

In short, all of this shipped together in M61:
https://storage.googleapis.com/chromium-find-releases-static/065.html#06592e080fd1cf4a188265ed4f9bcf826952671a

The only exceptions are scaleNonUniform which shipped in M73:
https://storage.googleapis.com/chromium-find-releases-static/57f.html#57f350e070536c7870cad53b3edf74f58cc51d50

And scaleNonUniformSelf isn't in the spec and was never in Chromium:
https://github.com/w3c/fxtf-drafts/issues/214
https://github.com/whatwg/compat/issues/19

The data for opera, opera_android, samsung_internet and webview_android
was mirrored.

### CODE REVIEW

**Comment by foolip** [comment]
Yep, I'm fixing that now, it revealed other issues too.

**Comment by foolip** [comment]
Alright, updated now. It became a pretty big change. I will squash into a single commit and update the description to match.

**Comment by vinyldarkscratch** [comment]
Oh no need to worry about squashing the commits on your end!  This repository is configured so that only squash merges are allowed, so it'll all be one commit anyways.  😉

**Comment by foolip** [comment]
> This repository is configured so that only squash merges are allowed, so it'll all be one commit anyways.

Yes, but that only concatenates commit messages in a pretty ugly format. Here I wanted to combine the two commit messages to tell a cleaner story about what has happened here.

**Comment by foolip** [comment]
I missed one thing: https://github.com/mdn/browser-compat-data/pull/6575
